### PR TITLE
LPS-133421 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133421

Hi Tim, I tried to change the user email address but I got an error to access the page and with the groupID. If it's really necessary change the user email please let me know.

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/controlmenu/SystemSettings.path [hunk 1]
```diff
@@ -47,8 +47,8 @@
 </tr>
 <!--ASSETS-->
 <tr>
-	<td>ASSETS_DYNAMIC_EXPORT_LIMIT</td>
-	<td>//div[contains(@data-field-name,'dynamicExportLimit')]//input[@type='text']</td>
+	<td>ASSETS_INPUT_TEXT_FIELD</td>
+	<td>//div[contains(@data-field-name,'${key_fieldLabel}')]//input[@type='text']</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/controlmenu/SystemSettings.path:51]: Unneeded

File: portal-web/test/functional/com/liferay/portalweb/macros/SystemSettings.macro [hunk 2]
```diff
@@ -14,12 +14,22 @@ definition {
...
 
+	macro configureDynamicSubscriptionLimit {
+		Type(
+			key_fieldLabel = "dynamicSubscriptionLimit",
+			locator1 = "SystemSettings#ASSETS_INPUT_TEXT_FIELD",
+			value1 = "${subscriptionLimitNumber}");
+
+		SystemSettings.saveConfiguration();
+	}
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/SystemSettings.macro:32]: Unneeded

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase [hunk 3]
```diff
@@ -1616,6 +1616,76 @@ definition {
...
+				portlet = "System Settings");
+		}
+
+		task ("Set the Dynamic Subscription Limit on Asset Publisher") {
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Assets",
+				configurationName = "Asset Publisher",
+				configurationScope = "System Scope");
+
+			SystemSettings.configureDynamicSubscriptionLimit(subscriptionLimitNumber = "0");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase:1637]: SystemSettings.editTextSetting

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase [hunk 4]
```diff
@@ -1616,6 +1616,76 @@ definition {
...
+				catch (Exception e) {
+				e.printStackTrace(out)
+				}
+			''';
+
+			ServerAdministration.executeScript(
+				language = "Groovy",
+				script = "${script}");
+		}
+	}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase:1687]: Add step to view email in Mock Mock

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase [hunk 5]
```diff
@@ -1616,6 +1616,76 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-126190. The user should receive an e-mail after subscribe to Asset Publisher when Dynamic Subscription Limit is set to 0."
+	@priority = "4"
+	test ViewSuccessMessageAfterForceSubscriptionWithGroovy {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase:1621]: ViewNotificationEmailWhenSetDynamicSubscriptionLimitToZero

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase [hunk 6]
```diff
@@ -1616,6 +1616,76 @@ definition {
...
+		property test.name.skip.portal.instance = "AssetPublisherUseCase#ViewSuccessMessageAfterForceSubscriptionWithGroovy";
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase:1623]: Add property to clear blacklist
Add property to enable smtp server

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update case according to comments ? Thanks

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update case according to comments ? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update case according to comments ? Thanks

Hi @Tim-Cao, PR updated.

File: portal-web/test/functional/com/liferay/portalweb/macros/asset/assetpublisher/AssetPublisherPortlet.macro [hunk 7]
```diff
@@ -327,7 +327,7 @@ definition {
 
 		Navigator.gotoNavTab(navTab = "Display Settings");
 
-		Panel.expandPanel(panel = "Set and Enable");
+		Panel.expandPanel(panel = "set-and-enable");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/asset/assetpublisher/AssetPublisherPortlet.macro:330]: Unneeded. This is a regression issue. https://issues.liferay.com/browse/LPS-136032

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase [hunk 8]
```diff
@@ -1521,6 +1521,91 @@ definition {
...
+				settingName = "Dynamic Subscription Limit",
+				settingValue = "0");
+
+			SystemSettings.saveConfiguration();
+		}
+
+		task ("Enable subscription in the Asset Publisher") {
+			ContentPagesNavigator.openViewContentPage(
+				pageName = "Asset Publisher Page",
+				siteName = "Test Site Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase:1554]:
```suggestion
			Navigator.gotoSitePage(
				pageName = "Asset Publisher Page",
				siteName = "Test Site Name");
```

File: portal-web/test/functional/com/liferay/portalweb/macros/asset/assetpublisher/AssetPublisherPortlet.macro (hunk 7 above)
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/macros/asset/assetpublisher/AssetPublisherPortlet.macro:330]: Thanks for letting me know! I will update the PR removing the change.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/29ff8b44919795768be072c9ad696f6fccbdd9f4...c6cf478944c572612d72c51cc6f39354d01b539d

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**